app/views/expenses/new.html.erb

```diff
@@ -1,14 +1,23 @@
-<h1>Expenses Form</h1>
+<section class='form-expense-container'>
+    <div class= 'form-expense-content'>
+        <h1>Expenses Form</h1>
+        
+        <%= form_with(model:@expense, url: expenses_path, local: true, class:'expense-form') do |form| %>
+        
+            <%= form.hidden_field :category_id, value: @category.id %>
+        
+            <div class="expense-field">
+                <%= form.label :name, 'Category', class: "" %>
+                <%= form.text_field :name %><br />
+            </div>
+        
+            <div class="expense-field">
+                <%= form.label :amount, 'Amount', class: "" %>
+                <%= form.number_field :amount %><br />
+            </div>
```

Comment:
- [ ] In your Expenses form it would be nice to add a back button so if a user clicks on the "Back" button (\<), the user navigates to the transactions(expenses) page for that category, this will create a better user experience so when users don't want to create a  transaction(expense)  they can click the back button 🤗 

- Good job adding validations to your forms 👍🏿 It would be nice to add a prompt when a user tries to submit an empty form, I see this response on my screen when I try to submit an empty expense form ⬇️ 

![Screenshot_238](https://github.com/Ayokunnumi1/wise-spender/assets/69638013/8b1e8388-12c7-43d5-a12a-fb949554156b)